modules/cql/src/blaze/elm/compiler/clinical_values.clj

```diff
@@ -102,8 +103,15 @@
 
 
 ;; 3.10. Ratio
-;;
-;; TODO
+(defmethod core/compile* :elm.compiler.type/ratio
+  [_ {:keys [numerator denominator]}]
+  (ratio/ratio (quantity/quantity (:value numerator) (cond
```

Comment:
It's better to use an `if` here, because there are only two branches.

Reply:
Done, thank you!